README.md

```diff
@@ -1,15 +1,15 @@
-<p align="center">
-    <img title="Laravel Zero" height="100" src="https://raw.githubusercontent.com/laravel-zero/docs/master/images/logo/laravel-zero-readme.png" />
+<p style="text-align: center; ">
+    <img title="Laravel Zero" height="100" src="https://raw.githubusercontent.com/laravel-zero/docs/master/images/logo/laravel-zero-readme.png" alt="Laravel Zero Logo" />
 </p>
 
-<p align="center">
-  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></img></a>
+<p style="text-align: center; ">
+  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></a>
   <a href="https://packagist.org/packages/laravel-zero/framework"><img src="https://img.shields.io/packagist/dt/laravel-zero/framework.svg" alt="Total Downloads"></a>
   <a href="https://packagist.org/packages/laravel-zero/framework"><img src="https://img.shields.io/packagist/v/laravel-zero/framework.svg?label=stable" alt="Latest Stable Version"></a>
   <a href="https://packagist.org/packages/laravel-zero/framework"><img src="https://img.shields.io/packagist/l/laravel-zero/framework.svg" alt="License"></a>
 </p>
 
-<h4> <center>This is a <bold>community project</bold> and not an official Laravel one </center></h4>
+<h4 style="text-align: center; ">This is a community project and not an official Laravel one</h4>
```

Comment:
Can you completely remove this line? 🤷🏻 It's true, but Nuno removed it from the main repository (so best to keep them in sync).

Reply:
I'm good with that if you guys are.